src/category_theory/reflect_isomorphisms.lean

```diff
@@ -0,0 +1,95 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+import category_theory.limits.limits
+import category_theory.limits.preserves
+
+open category_theory category_theory.limits
+
+namespace category_theory
+
+universes v₁ v₂ u₁ u₂
+
+variables {C : Type u₁} [𝒞 : category.{v₁} C]
+include 𝒞
+
+section reflects_iso
+variables {D : Type u₂} [𝒟 : category.{v₂} D]
+include 𝒟
+
+/--
+Define what it means for a functor `F : C ⥤ D` to reflect isomorphisms: for any
+morphism `f : A ⟶ B`, if `F.map f` is an isomorphism then `f` is as well.
+Note that we do not assume or require that `F` is faithful.
+-/
+class reflects_isomorphisms (F : C ⥤ D) :=
+(reflects : Π {A B : C} (f : A ⟶ B) [is_iso (F.map f)], is_iso f)
+
+-- TODO: reflects iso is equivalent to reflecting limits of shape 1 (punit)
```

Comment:
I'll assume that your intention is to leave this TODO in mathlib. (Which is fine!)

Reply:
Yeah!